src/styleguide/css/pattern-scaffolding.css

```diff
@@ -1,65 +1,185 @@
 /* Color palette previews */
 .sg-pattern-sub-heading {
-  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  margin: 0 0 1rem;
+  padding: 0;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
   font-size: 0.85rem;
   line-height: 1;
-  margin: 0 0 1rem 0;
-  padding: 0;
 }
 
 /* Color palette previews */
+
+/** @define sg-colors */
 .sg-colors {
-  padding: 0;
   display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
-}
-
-.sg-colors__color {
-  list-style-type: none;
-  width: 20%;
-  min-width: 12rem;
-  box-sizing: border-box;
-  padding: 1rem;
   margin: 0;
+  padding: 0;
+  flex-wrap: wrap;
+  list-style: none;
 }
 
-.sg-colors__swatch,
-.sg-colors__name,
-.sg-colors__var {
-  display: block;
-}
-
-.sg-colors__swatch {
-  height: 4rem;
-  margin-bottom: 0.5rem;
+.sg-colors > li {
+  margin: 0 0.5em 0.5em 0;
+  padding: 0.3em;
+  border: 1px solid #ddd;
+  flex: auto;
+  max-width: 14em;
+  min-width: 5em;
 }
 
-.sg-colors__name {
-  font-family: sans-serif;
-}
-
-
 /* Color palette previews */
 .sg-fonts {
   padding: 0;
 }
 
 .sg-fonts__font {
+  margin: 0 0 2rem;
   list-style-type: none;
-  margin: 0 0 2rem 0;
 }
 
 .sg-fonts__weight {
-  font-family: sans-serif;
   display: block;
+  font-family: sans-serif;
 }
 
 .sg-fonts__css {
   display: block;
-  margin: 0 0 0.5rem 0;
+  margin: 0 0 0.5rem;
 }
 
 .sg-fonts__sample {
   margin: 0 0 0.5rem 2rem;
 }
+
+/** @define sg-swatch */
+.sg-swatch {
+  display: block;
+  height: 6em;
+  margin-bottom: 0.3em;
+}
+
+/** @define sg-swatch-check */
+.sg-swatch-check {
+  display: grid;
```

Comment:
CSS Grid. Yay!

_But_, is it not overkill in this instance, since it's a 1-dimensional layout? Flexbox could achieve the same thing.

Happy to accept this as it is, but if you fancy it/have spare time it might be worth considering simplifying this to be flex-based instead. Ditto for the nested `.sg-swatch-check__group` list.

Reply:
I'm actually thinking of iterating a bit more on this so we can have a proper look on large display so it shows all the colours on the same row. Flexbox is a bit of a faff when it comes to that, while on the `.sg-swatch-check__group` is enough.
Let me see if I can get something done soon enough.